### ISSUE TITLE
Dropout on inference

### ISSUE THREAD

**Issue by kevinzezel** [tae898/age-gender#16]
First of all congratulations for the work.

I ran app.py through flask and did some tests. I noticed that with each run with the same photo the result changes a little. Looking at the source code I noticed that the dropout is enabled. Why did you enable dropout in inference? Can I disable it to always generate constant results for the same image?

```
for model_ in ["age", "gender"]:
    model = ResMLP(**read_json(f"./models/{model_}.json")["arch"]["args"])
    checkpoint = f"models/{model_}.pth"
    checkpoint = torch.load(checkpoint, map_location=torch.device(device))
    state_dict = checkpoint["state_dict"]
    model.load_state_dict(state_dict)
    model.to(device)
    model.eval()
    # enable_dropout(model) **Can I safely comment out this line?**
```

**Comment by tae898** [comment]
Dropout is enabled to allow “Monte Carlo dropout” to estimate the uncertainty. Indeed, this is the key of my work. Please check out the paper! :)
